### ISSUE TITLE
DDNS Support

### ISSUE THREAD

**Issue by DimZ2** [AdguardTeam/AdGuardHome#4634]
Dear developes, please, add DDNS support, as in AdGuard DNS. Thank you :)

**Comment by lucktu** [comment]
Yes, this function is very important. hope it can be realized soon.

**Comment by Dimtika** [comment]
@ainar-g say please, do you know, how soon this feature will be released?

**Comment by ainar-g** [comment]
No plans currently.
